packages/angular/src/generators/init-mfe/files/webpack.config.js__tmpl__

```diff
@@ -0,0 +1,41 @@
+const ModuleFederationPlugin = require("webpack/lib/container/ModuleFederationPlugin");
+const mf = require("@angular-architects/module-federation/webpack");
+const path = require("path");
+
+const sharedMappings = new mf.SharedMappings();
+sharedMappings.register(path.join(__dirname, "tsconfig.json"), [
```

Comment:
The [`register` implementation](https://github.com/angular-architects/module-federation-plugin/blob/main/libs/mf/src/utils/shared-mappings.ts#L21) just reads the tsconfig file and uses the defined `paths` in it. The `tsconfig.json` of an app in Nx workspaces have no `paths` defined, they are extended from the `tsconfig.base.json` at the root. Generating this with `tsconfig.json` should always register an empty array because no `paths` would be found in the config.

Reply:
If they extend the base `tsconfig.json` will they not inherit that paths property?

Reply:
Yes, but that's only if you read it using the TS utilities or any similar that actually process the `extends`. Using `fs.readFileSync` and `JSON5.parse` just read the json file and parse the string to json respectively, it won't bring the properties from the extended configuration.

Reply:
Oh the underlying implementation of that package doesn't get the inherited properties... That's really interesting. Ok, I'll need to make a change to this. Good catch!